src/software/new_geom/line.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <stdexcept>
+
+#include "software/new_geom/polynomial.h"
+
+class Line final : public Polynomial
+{
+   public:
```

Comment:
Can we also add a `slope()` function to get the slope of the line?

Reply:
The slope can be accessed at the moment using the method inherited from `Polynomial`, `getCoeff()`. If we are adding `slope()` to this class, should we also add a `yIntercept()` function? And what about setters?

Reply:
And #1024 might be a bit relevant here, is there a convention for when to prefix a getter with `get`?

Reply:
I agree with Mat that we should have a `getSlope` function, and I think you make a good point about a `getYIntercept` as well (might as well have both). #1024 is in my backlog, but I think that for this class, where we won't call the methods _that_ often, we should call them `getSlope` and `getYIntercept`.

Reply:
Addressed in 31c130a36c3b283ceeb3119c01c817db7f16bde2, `Line` class no longer has `getCoeff` and `setCoeff` methods, but instead getters and setters for slope and y-intercept.